core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -228,6 +228,15 @@ public enum Property {
       "2.1.0"),
   GENERAL_RPC_TIMEOUT("general.rpc.timeout", "120s", PropertyType.TIMEDURATION,
       "Time to wait on I/O for simple, short RPC calls", "1.3.5"),
+  GENERAL_FILENAME_PREFIX("general.filename.", null, PropertyType.PREFIX,
+      "Properties related to filename allocations.", "2.1.3"),
+  GENERAL_FILENAME_BASE_ALLOCATION("general.filename.base.allocation", "100", PropertyType.COUNT,
+      "The minimum number of filenames that will be allocated from Zookeeper at a time.", "2.1.3"),
+  GENERAL_FILENAME_JITTER_ALLOCATION("general.filename.jitter.allocation", "100",
+      PropertyType.COUNT,
+      "The size of the jitter that will be applied to the `general.filename.base.allocation` when allocating "
+          + "filenames from Zookeeper. This will result in an allocation between base and (base + jitter).",
```

Comment:
```suggestion
          + "filenames from Zookeeper. This will result in an allocation between base and (base + jitter).  This property is ignored when its <= 0 and only base is used.",
```